docs/src/trace-viewer.md

```diff
@@ -52,13 +52,14 @@ await page.goto('https://playwright.dev');
 await context.tracing.stop({ path: 'trace.zip' });
 ```
 
-You can also use `trace: 'retain-on-failure'` if you do not enable retries but still want traces for failed tests.
-
 Available options to record a trace:
+- `'on-first-retry'` - Record a trace only when retrying a test for the first time.
```

Comment:
Mention to make sure to have retries >= 1?

Reply:
This doc needs a full rework which will be part of another pr